samples/cancel-custom-job.js

```diff
@@ -47,8 +47,7 @@ async function main(customJobId, project, location = 'us-central1') {
     // Cancel custom job request
     const [response] = await jobServiceClient.cancelCustomJob(request);
 
-    console.log('Cancel custom job response');
-    console.log(`${response}`);
+    console.log('Cancel custom job response:\n', response);
```

Comment:
Nit: why are you printing the two parts on separate lines as apposed to 1 line ? 

For example if you wanted to print it on one line you could do 

``` `Cancel custom job response: ${response}` ```